docs/content_management/pages/page_builder_guide.md

```diff
@@ -0,0 +1,213 @@
+
+![Drop zone line](drop_zone_line.png)
+
+Ready-to-use blocks available in [[= product_name =]] have their own, unique functions, but you can also [add your own, custom blocks](create_custom_page_block.md). All available tools and settings, that Page Builder comes with, enable you to customize the content appearing on the page.
+
+You can check all ready-to-use blocks available in Page Builder in User Documentation, [Block reference page](https://doc.ibexa.co/projects/userguide/en/latest/content_management/block_reference/).
+
+#### Work with blocks
+
+Working with blocks is intuitive. You don't have to worry about placing blocks in the proper place from the start - you can reorder the at any time.
+You can reorder blocks in a few ways:
+
+- drag and drop block in the desired location on a drop zone
+- select block and use up and down arrow on the keyboard
+- access Structure view and use 'Move up' and 'Move down' function in the settings of the block or drag and drop to change the position in the structure
+
+![Structure view - drag and drop](structure_view_drag_drop.png)
+
+You can manage each block by accessing its settings. To do it, click settings icon next to the block's name.
+
+![Block settings](block_settings.png)
+
+Available settings are:
+
+- Move up - allows you to change position of the block on the page by moving it up
+- Move down - allows you to change position of the block on the page by moving it down
+- Configuration - allows you to access configuration window
+- Duplicate - duplicates existing block by creating its copy
+- Refresh - refreshes preview of the block
+- Delete - deletes existing block
+
+#### Schedule content
+
+Page Builder comes with a Scheduler - is a tool that allows you to schedule content publication.
+
+You can schedule to be published, or hidden in Page Builder in two ways with:
+
+- **Scheduler tab** - it's available in the configuration of all Page blocks. In this tab you can set the date and time when the block becomes visible and when it disappears from a Page.
+
+![Scheduler tab](scheduler_tab.png)
+
+- **Content Scheduler** - it's one of the block available in Page Builder Elements menu. To proceed with the schedule, go to **Basic** tab of the block, then click **Select contennt** and confirm your choice, and set date and time in the Content airtime settings window.
+
+![Content Scheduler](content_scheduler.png)
+
+For more information, see [Schedule publication](https://doc.ibexa.co/projects/userguide/en/latest/content_management/schedule_publishing/).
+
+## Benefits
+
+### Manage your pages without technical skills
+
+Thanks to intuitive and plain Page Builder interface, you can create and manage your website without the need of having advanced technical skills.
+Drag-and-drop elements toolbar, separated page zones and transparent sections - these are the elements that make working with Page Builder really intuitive and quick.
```

Comment:
```suggestion
Drag-and-drop elements toolbar, separated page zones and transparent sections - these are the elements that make working with Page Builder really intuitive and quick.
```
Drag-and-drop elements toolbar - sounds a lot like a command. Maybe just "The **Elements** toolbar"
"separated" - as in "well defined" or "visible"?
"transparent sections" - in what sense? What do you mean by a section here?